fastlane/Fastfile

```diff
@@ -708,3 +684,61 @@ platform :android do
     end
   end
 end
+
+def release_version_current
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  VERSION_FORMATTER.release_version(current_version)
+end
+
+def release_version_next
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  release_version_next = VERSION_CALCULATOR.next_release_version(version: current_version)
+  VERSION_FORMATTER.release_version(release_version_next)
+end
+
+# Compute the version to be used by the code freeze step in the release process.
+#
+# It first increments the minor number, which also resets the build number to 0.
+# It then bumps the build number so the -rc-1 can be appended to the code freeze version.
+def release_version_for_code_freeze
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  next_version = VERSION_CALCULATOR.next_release_version(version: current_version)
+  beta_version_first = VERSION_CALCULATOR.next_build_number(version: next_version)
+  VERSION_FORMATTER.beta_version(beta_version_first)
+end
+
+def release_version_next_beta
+  current_version = VERSION_FORMATTER.parse(VERSION_FILE.read_version_name)
+  beta_version_next = VERSION_CALCULATOR.next_build_number(version: current_version)
+  VERSION_FORMATTER.beta_version(beta_version_next)
+end
+
+def build_code_current
+  build_code_current = VERSION_FILE.read_version_code
+  BUILD_CODE_FORMATTER.build_code(build_code: build_code_current)
+end
+
+def build_code_next
+  build_code_current = VERSION_FILE.read_version_code
+  build_code_next = BUILD_CODE_CALCULATOR.next_build_code(build_code: build_code_current)
+  BUILD_CODE_FORMATTER.build_code(build_code: build_code_next)
+end
+
+def commit_version_bump
+  git_commit(
+    path: VERSION_PROPERTIES_PATH,
+    message: 'Bump version name and code',
+    allow_nothing_to_commit: false
+  )
+end
+
+def release_branch_name(release_version: release_version_current)
+  "#{RELEASE_BRANCH_ROOT}#{release_version}"
+end
+
+def ensure_git_branch_is_release_branch
+  # Verify that the current branch is a release branch. Notice that `ensure_git_branch` expects a RegEx parameter
+  ensure_git_branch(branch: "^#{RELEASE_BRANCH_ROOT}")
+end
+
+RELEASE_BRANCH_ROOT = 'release/'
```

Comment:
This const _could_ go at the top with all the others, but since it's used only by the two release branch methods, it felt clearer to put it at the bottom here, after its consumers.

It's a bit inconsistent but I think it works.

Happy to move it up with the rest if you think it's not worth breaking the pattern.